test/model/rotbaum/test_model.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+# Third-party imports
+import numpy as np
+from itertools import chain
+
+# First-party imports
+from gluonts.model.rotbaum import TreePredictor
+
+
+def test_accuracy(accuracy_test, dsinfo):
+    def quantile_loss(true, pred, quantile):
```

Comment:
Can you replace with using the built-in quantile loss calculation to prevent duplicate code.

Reply:
For instance try to replace this test with the standard `test_accuracy`:
```
@pytest.mark.parametrize("quantiles", [[0.1, 0.5, 0.9], [0.5]])
@pytest.mark.parametrize("hybridize", [True, False])
def test_accuracy(
    Estimator, accuracy_test, hyperparameters, hybridize, quantiles
):
    hyperparameters.update(
        num_batches_per_epoch=100, hybridize=hybridize, quantiles=quantiles
    )

    accuracy_test(Estimator, hyperparameters, accuracy=0.20)
```
as an example from `test/model/seq2seq/test_model.py`

Reply:
Postponed for now, as discussed offline.